pkg/apis/admissionregistration/types.go

```diff
@@ -326,4 +324,9 @@ type ServiceReference struct {
 	// this service.
 	// +optional
 	Path *string
+
+	// If specified, the port on the service that hosting webhook.
+	// `port` should be a valid port number (1-65535, inclusive).
+	// +optional
+	Port int32
```

Comment:
use `*int32` in internal packages

Reply:
> is using int32 intended?

it is. external versions use `*int32`, and default `nil` to `443`, and internal versions never encounter nil

Reply:
sounds good. I thought we have fuzz tests that don't go through defaulting and test roundtripping